### DISCUSSION TITLE
Model Driven App with Custom Pages - Solution Unpack - issue - PA3999 - Unable to parse templat...

### DISCUSSION THREAD

**Discussion by wojtasey** [microsoft/PowerApps-Language-Tooling#685] [Q&A]
Hi Team we are experiencing issue when trying to unpack Model Driven App solution using below command

_pac solution unpack --zipFile C:\ModelDrivenAppSolution.zip --folder C:\ModelDrivenAppSolution\ --packageType Unmanaged --localize false --errorlevel Verbose --singleComponent None --useLcid false --processCanvasApps true --allowDelete true --allowWrite true --clobber true_

PA3999 - Unable to parse template file Microsoft_CoreControls_RadioGrouptemplate_dataField

![image (2)](https://github.com/microsoft/powerplatform-build-tools/assets/87713372/0e62b8a8-0975-4b68-8f15-73a87161287a)

**Comment by devkeydet** [comment]
Transferring to https://github.com/microsoft/PowerApps-Tooling since the --processCanvasApp uses the experimental preview library from this repo.

**Comment by marcelbf** [comment]
To source control your canvas app, please use the [Power Platform git integration](https://learn.microsoft.com/power-platform/alm/git-integration/canvas-apps-git-integration).

More information about the [YAML serialization format](https://learn.microsoft.com/power-apps/maker/canvas-apps/power-apps-yaml#power-apps-yaml-schema-versions) is available in the documentation.
